opbeat/events.py

```diff
@@ -15,7 +15,7 @@
 from opbeat.utils import varmap
 from opbeat.utils.encoding import shorten, to_unicode
 from opbeat.utils.stacks import get_stack_info, iter_traceback_frames, \
-                               get_culprit
+    get_culprit
```

Comment:
while you're pep8in', how about using `from ... import ()` instead of the trailing backslash?